spec/reek/ast/object_refs_spec.rb

```diff
@@ -2,57 +2,16 @@
 require_lib 'reek/ast/object_refs'
 
 RSpec.describe Reek::AST::ObjectRefs do
-  let(:refs) { Reek::AST::ObjectRefs.new }
+  let(:refs) { described_class.new }
```

Comment:
I find the fact that `subject` is actually `described_class.new` automatically a little confusing.